### PR TITLE
Fix pending after first message

### PR DESCRIPTION
**Pull request by luke-cha**
### Details
Add detailed waiting logic for the issue that Bot's message is pending after sending the first message

### How to
Add logic to wait until the last message to the group channel is not null (up to 3 seconds) because it determines that an error occurs when the channel loads immediately after sending a message to the group channel

### CODE REVIEW

File: src/hooks/useCreateGroupChannel.ts [hunk 1]
```diff
@@ -49,8 +49,12 @@ export function useCreateGroupChannel(
           message: firstMessage,
         });
       }
-      // let's wait for a second to make sure channel is created & ready
-      await delay(1000);
+      let count = 0;
+      while (groupChannel.lastMessage == null && count < 30) {
+        await delay(100);
+        count += 1;
+      }
```
Comment (AhyoungRyu) [inline src/hooks/useCreateGroupChannel.ts]: nit;

```
async function waitForLastMessage(groupChannel, maxRetries = 30, retryInterval = 100) {
  let count = 0;
  while (groupChannel.lastMessage == null && count < maxRetries) {
    await delay(retryInterval);
    count++;
  }
  await delay(500);
}
```
How about having this logic separately and call it in here? 
```
// usage 
await waitForLastMessage(groupChannel);
```

Comment (luke-cha) [reply src/hooks/useCreateGroupChannel.ts]: @AhyoungRyu Your suggestion is also good. Could you update it?

Comment (AhyoungRyu) [reply src/hooks/useCreateGroupChannel.ts]: Yeah sure will do.

**Review by AhyoungRyu — APPROVED** [review]
Thank you, Luke!

**Pull request merged**